### DISCUSSION TITLE
vue-tips/an-url-object-working-on-both-server-side-and-client-side

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#11] [Announcements]
# vue-tips/an-url-object-working-on-both-server-side-and-client-side

Nuxt provides a helper function called useRequestURL that returns an URL object working on both server side and client side.

https://mokkapps.de/vue-tips/an-url-object-working-on-both-server-side-and-client-side

**Comment by sylvio-cavalcanti** [comment]
This is just what I needed for getting the full url on the server side, thank you very much!
